src/components/scene/embedded.js

```diff
@@ -0,0 +1,13 @@
+var register = require('../../core/component').registerComponent;
+
+module.exports.Component = register('embedded', {
+  schema: { default: true },
+
+  update: function () {
+    if (this.data === true) {
+      this.el.removeFullScreenStyles();

```

Comment:
It might be more clear if an `embedded` component called `addEmbeddedStyles`/`removeEmbeddedStyles`. Right now, the code has a few double-negatives.